### ISSUE TITLE
choose network

### ISSUE THREAD

**Issue by joy31-eng** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#63]
Hello, I would like to ask a question, when I use unet, it is normal, but switch to other network, the loss of training set is decreased, but the loss of verification set has not decreased, and dice is 0

**Comment by SmileMood** [comment]
hello,I get the same  problem,could you tell me how you solve this problem?

**Comment by bigmb** [comment]
It can be due to model not learning anything while using any RNN type model. |
Does this happen while using nested Unet?

**Comment by SmileMood** [comment]
when i using unet  and trans-unet model, it take such problem.
how can i solve this problem?

**Comment by SmileMood** [comment]
Thanks for your reply.
when I using unet  and trans-unet model, it take such problem.
how can I solve this problem?

**Comment by bigmb** [comment]
Which model is trans-Unet model? 
Can you explain more about the data and transformation you are using?

**Comment by SmileMood** [comment]
I'm sorry,it should be attention unet.
I found my error from data_transform,because when  I using my own data,I changed this by mistake.
Now the dice score isn't 0,but it still very low.

**Comment by xike-lk** [comment]
hello,I get the same problem,could you tell me how to slove this problem?

**Comment by bigmb** [comment]
Normally, RNN and attention modules in the unet architecture are data hungry.
Do you get the same results in nested unet? 
( RNN architecture is not suitable for most task.)

**Comment by xike-lk** [comment]
> Normally, RNN and attention modules in the unet architecture are data hungry. Do you get the same results in nested unet? ( RNN architecture is not suitable for most task.)

Thank you for replying ! I have solved this problem ! I have another question . I want to do multi-classes tasks , where should I edit the code？

**Comment by bigmb** [comment]
Change the loss the function according to your task. (its sigmoid now - calc loss) 
Look into model loading in pytorch_run.
